lib/PrintAsObjC/DeclAndTypePrinter.cpp

```diff
@@ -393,7 +393,8 @@ class DeclAndTypePrinter::Implementation
         os << " SWIFT_COMPILE_NAME(\"" << Elt->getName() << "\")";
       }
 
-      if (auto ILE = cast_or_null<IntegerLiteralExpr>(Elt->getRawValueExpr())) {
+      auto *RVE = Elt->getStructuralRawValueExpr();
+      if (auto ILE = cast_or_null<IntegerLiteralExpr>(RVE)) {
```

Comment:
The "or null" here was to not print values not explicitly provided by the user, but I guess that only works when printing from source anyway. Might as well drop it, or check for `isImplicit` instead (which probably isn't preserved through serialization).